### PR TITLE
Fix IllegalArgumentException from event with unnamed parameter

### PR DESCRIPTION
**Pull request by dizk**
### What does this PR do?
When trying to wrap TruffleJson contracts containing events with unnamed parameters javapoet complain about `not a valid name: ` and throws an IllegalArgumentException.

### Where should the reviewer start?
`codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java` contains the fix
`codegen/src/test/resources/truffle/ModifiedMetaCoin/MetaCoin.sol` contains an event with unnamed parameters.

### Why is it needed?
It is needed to generate wrappers for contracts declaring events with unnamed parameters. 

Exception thrown:
```
Exception in thread "main" java.lang.IllegalArgumentException: not a valid name: 
	at com.squareup.javapoet.Util.checkArgument(Util.java:64)
	at com.squareup.javapoet.FieldSpec.builder(FieldSpec.java:92)
	at com.squareup.javapoet.TypeSpec$Builder.addField(TypeSpec.java:499)
	at org.web3j.codegen.SolidityFunctionWrapper.buildEventResponseObject(SolidityFunctionWrapper.java:1527)
	at org.web3j.codegen.SolidityFunctionWrapper.buildEventFunctions(SolidityFunctionWrapper.java:1694)
	at org.web3j.codegen.SolidityFunctionWrapper.buildFunctionDefinitions(SolidityFunctionWrapper.java:382)
	at org.web3j.codegen.SolidityFunctionWrapper.generateJavaFiles(SolidityFunctionWrapper.java:220)
	at org.web3j.codegen.SolidityFunctionWrapper.generateJavaFiles(SolidityFunctionWrapper.java:185)
	at org.web3j.codegen.TruffleJsonFunctionWrapperGenerator.generate(TruffleJsonFunctionWrapperGenerator.java:152)
	at org.web3j.codegen.TruffleJsonFunctionWrapperGenerator.main(TruffleJsonFunctionWrapperGenerator.java:112)
```

### CODE REVIEW

**Comment by dizk** [comment]
I think this might be related to aswell web3j/web3j-cli#58

File: codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/ConvertLib.json [hunk 1]
```diff
@@ -0,0 +1,548 @@
+{
```
Comment (SweeXordious) [inline codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/ConvertLib.json:1]: Can you please shrink the content into one line please ?

File: codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/MetaCoin.json [hunk 2]
```diff
@@ -0,0 +1,2740 @@
+{
```
Comment (SweeXordious) [inline codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/MetaCoin.json:1]: Same, Can you please shrink the content into one line please ?

File: codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/Migrations.json [hunk 3]
```diff
@@ -0,0 +1,966 @@
+{
```
Comment (SweeXordious) [inline codegen/src/test/resources/truffle/ModifiedMetaCoin/build/contracts/Migrations.json:1]: Same, Can you please shrink the content into one line please ?

**Review by SweeXordious — APPROVED** [review]
LGTM. Some comments to be addressed

**Comment by dizk** [comment]
@SweeXordious I have shrinked the json files. (No access to my computer at the moment so I had to do it in 3 commits via browser 😅)

**Pull request closed without merging**